diff -Nur -x '*.orig' -x '*.rej' e-smith-grub-1.0.0/root/etc/e-smith/templates/boot/grub/grub.conf/UpdateSplash mezzanine_patched_e-smith-grub-1.0.0/root/etc/e-smith/templates/boot/grub/grub.conf/UpdateSplash
--- e-smith-grub-1.0.0/root/etc/e-smith/templates/boot/grub/grub.conf/UpdateSplash	1969-12-31 17:00:00.000000000 -0700
+++ mezzanine_patched_e-smith-grub-1.0.0/root/etc/e-smith/templates/boot/grub/grub.conf/UpdateSplash	2007-07-13 13:44:31.000000000 -0600
@@ -0,0 +1,15 @@
+{
+    our @newlines;
+    foreach( @lines ) {
+	next if /^foreground/ || /^background/;
+	if ( m#^splashimage=(.*)/# ) {
+	    push @newlines, "splashimage=$1/smeserver.xpm.gz";
+	    push @newlines, "foreground 000000";
+	    push @newlines, "background 4E95D3";
+	} else {
+	    push @newlines, $_;
+	}
+    }
+    @lines = @newlines;
+    $OUT = "";
+}
