diff -Nur -x '*.orig' -x '*.rej' e-smith-grub-0.0.1/root/etc/e-smith/events/actions/grub-install-raid mezzanine_patched_e-smith-grub-0.0.1/root/etc/e-smith/events/actions/grub-install-raid
--- e-smith-grub-0.0.1/root/etc/e-smith/events/actions/grub-install-raid	2006-02-08 12:53:05.056108640 +1100
+++ mezzanine_patched_e-smith-grub-0.0.1/root/etc/e-smith/events/actions/grub-install-raid	2006-02-08 12:52:49.464821349 +1100
@@ -32,7 +32,7 @@
 
 echo "Calling grub-install on $HDZERO"
 
-grub --batch <<HERE
+grub --no-floppy --batch <<HERE
 device (hd0) $HDZERO
 root (hd0,0)
 setup (hd0)
@@ -48,7 +48,7 @@
 
 echo "Calling grub-install on $HDONE"
 
-grub --batch <<HERE
+grub --no-floppy --batch <<HERE
 device (hd0) $HDONE
 root (hd0,0)
 setup (hd0)
